### PR TITLE
Add initial drop of non-default Leap 16 wallpapers

### PR DESCRIPTION
**Pull request by lkocman**
* Factory will simply use latest Leap wallpaper drop we intend to suggest package from branding-openSUSE package
* Gecko from Photo competition gh#openSUSE/wallpapers#18
* Gradient with the new lcp's project logo gh#openSUSE/branding#156
* Install also files from previous versions (will be split into subpackages in the .spec)
* Update Readme

### CODE REVIEW

**Comment by lkocman** [comment]
Test build https://build.opensuse.org/package/show/home:lkocman:branches:X11:common:Factory/wallpapers-openSUSE-extra

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,19 @@
-install: 
-	mkdir -p /usr/share/gnome-background-properties/
-	install -D -m 0644 wallpapers-leap15.xml ${DESTDIR}/usr/share/gnome-background-properties/wallpapers-leap15.xml
-	mkdir -p /usr/share/wallpapers/leap15/
-	install -D -m 0644 leap15/*.jpg ${DESTDIR}/usr/share/wallpapers/leap15/
-	install -D -m 0644 leap15/*.xml ${DESTDIR}/usr/share/wallpapers/leap15/
+VERSION = 16
+SVG_FILES = $(wildcard leap$(VERSION)/*.svg)
```
Comment (hellcp) [inline Makefile]: How about we install both directories and split them in the spec?

Comment (lkocman) [reply Makefile]: We could split the older ones to the -retro subpackage.

Comment (lkocman) [reply Makefile]: But it will *significantly* increase build time with optipng -o5 ... unless we don't do that on build time.

**Comment by lkocman** [comment]
Merging and submitting https://build.opensuse.org/request/show/1205517

**Pull request merged**